lib/bugsnag/api/error.rb

```diff
@@ -59,7 +57,9 @@ def data
       end
 
       def response_error
-        "Error: #{data[:error]}" if data.is_a?(Hash) && data[:error]
+        return nil unless data.is_a?(Hash)
+        return "Error: #{data[:error]}" if data[:error]
+        "Errors: #{data[:errors]}" if data[:errors]
```

Comment:
Alternatives which break complete backwards compat for message parsing would be:
```
errors = Array(data[:errors] || data[:error])
"Errors: #{errors}" if errors.any?
```
or
```
errors = data[:errors] || (data[:error] ? [data[:error]] : nil)
"Errors: #{errors}" if errors.any?
```

Reply:
We may also want to pretty print lists of errors in some manner rather than relying on `Array::to_s`.